docs/public-networks/how-to/use-configuration-file/profile-file.md

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_position: 1
+---
+
+# Use a profile file
+
+Besu provides pre-configured profile files containing custom settings for some common use cases.
+You can load these profile files using the [`--profile`](../../reference/cli/options.md#profile) CLI option.
+For example:
+
+```bash
+besu --config-file=config.toml --profile=staker
+```
+
+In this example, `config.toml` is the user-provided [configuration file](index.md), and `staker` is
+the pre-configured profile containing custom settings.
+
+Any configuration options explicitly set in the configuration file or command line will overwrite
+the same options set in the profile file.
+See the [configuration order of precedence](index.md#configuration-order-of-precedence) for more information.
+
+You can use the following profiles:
+
+- [Minimalist staker profile](#minimalist-staker-profile)
+- [Staker profile](#staker-profile)
+- [Enterprise/Private profile](#enterpriseprivate-profile)
+
+## Minimalist staker profile
+
+For stakers who want to maximize their hardware value but don't want to serve full sets of data to
+their peers, Besu provides a minimalist staker profile.
+See the
+[minimalist staker profile file](https://github.com/hyperledger/besu/blob/8b64023a121ea996ef60e4b7e2299c5807683f90/config/src/main/resources/profiles/minimalist-staker.toml)
+for the custom settings.
+
```

Comment:
I wonder if there's a way in which we could actually display contents of the toml file?

For example: https://github.com/hyperledger/besu/blob/8b64023a121ea996ef60e4b7e2299c5807683f90/config/src/main/resources/profiles/enterprise-private.toml

So users can see what the values are, and minimise maintenance for us if the file changes.

Reply:
I can't quite figure out an easy way to do this in Docusaurus. I tried [this solution](https://stackoverflow.com/questions/28338017/is-there-a-way-to-embed-github-code-into-an-iframe) and it caused some errors. Maybe something to solve in the future as this need arises again.

Reply:
Yeah GitHub probably doesn't allow content to be embedded in iFrames.